server/resourcemanager/resource_manager_test.go

```diff
@@ -20,38 +20,76 @@ func TestSampleResource(t *testing.T) {
 		SomeValue: "the value",
 	}
 
-	secondSample := sampleResource{
-		ID:        "2",
-		Name:      "the name 2",
-		SomeValue: "the value 2",
-	}
-
-	thirdSample := sampleResource{
-		ID:        "3",
-		Name:      "the name 3",
-		SomeValue: "the value 3",
-	}
-
 	sampleUpdated := sampleResource{
 		ID:        "1",
 		Name:      "the name updated",
 		SomeValue: "the value updated",
 	}
 
+	prepareSortByID := func(m *sampleResourceManager) {
```

Comment:
this code is unrelated to this PR directly, but I realized the behaviour of the sort test was incorrect. the order could be coming from any field, not the specific one being tested. While this is not very relevant for this specific tests, 
this test serves as a "meta" test for the test suit itself. This guarantees that the test works as expected when actual resources are implemented.